verification/src/compiler/version.rs

```diff
@@ -99,6 +104,29 @@ pub enum CompilerVersion {
     Nightly(NightlyVersion),
 }
 
+impl CompilerVersion {
+    fn version(&self) -> &Version {
+        match self {
+            CompilerVersion::Nightly(v) => &v.version,
+            CompilerVersion::Release(v) => &v.version,
+        }
+    }
+
+    fn date(&self) -> Option<NaiveDate> {
```

Comment:
```suggestion
    fn date(&self) -> Option<&NaiveDate> {
```
Why not to return just a reference?